dali/base/dadfs.cpp

```diff
@@ -7684,7 +7701,7 @@ bool CDistributedFileDirectory::exists(const char *_logicalname,IUserDescriptor
 
 bool CDistributedFileDirectory::existsPhysical(const char *_logicalname, IUserDescriptor *user)
 {
-    Owned<IDistributedFile> file = lookup(_logicalname, user, false, false, false, NULL, defaultTimeout);
+    Owned<IDistributedFile> file = lookup(_logicalname, user, false, false, false, NULL, true, defaultTimeout);
```

Comment:
is privilegedUser=true always correct here, or is this is a stop gab, before tightening up later?

Reply:
Yes it's always true because just checking if it exists shouldn't cause an sensitive information to be read.  Add comment in code about this.